.github/workflows/deployAzureBackend.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy Backend Azure Resources
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - "backend/**"
+
+env:
```

Comment:
If you're passing them in as command-line arguments anyway, I would skip the environment setup and just include them in the `run` command:

    docker login ${{secrets.ACR_REPO_URL}} -u ${{secrets.ACR_ADMIN_USERNAME}} etc.